src/components/task/crumb-mixin.cjsx

```diff
@@ -30,6 +30,17 @@ module.exports =
 
     doesAllowSeeAhead or index <= latestIndex
 
+  _buildSectionLabel: (chapter_section, crumbs) ->
+    chapterSection = _.clone(chapter_section)
+
+    # ignore 0 in chapter sections
```

Comment:
where the 0 is ignored..? I am unsure..seems like that should be the case everywhere right?  who should I ask?